### PR TITLE
Update Usage docs

### PR DESCRIPTION
**Pull request by mgrauer**

### CODE REVIEW

File: docs/USAGE.rst [hunk 1]
```diff
@@ -115,17 +115,17 @@ An image is copied into the ``Original`` folder before it will be redacted and w
...
 - "Quarantine" for more reprocessing, sending it to the ``Quarantined`` folder (go to step 4a)
 
 Step 4a Users Contact IMS and Reprocess for Potential Resolution
 ----------------------------------------------------------------
 
-The ``Quarantined`` folder is for holding images that may hold PHI/PII. Users should contact IMS for any WSI files in the ``Quarantined`` folder, and then review files for reprocessing based on instructions from IMS. If PHI/PII is seen in an image or metadata field, that is somewhere other than the ``Available To Process`` folder, it should be quarantined for reprocessing.
+The ``Quarantined`` folder is for holding images that may hold PHI/PII. Users should contact IMS via Squish (https://www.squishlist.com/seerdms/support/) for any WSI files in the ``Quarantined`` folder, and then review files for reprocessing based on instructions from IMS. If PHI/PII is seen in an image or metadata field, that is somewhere other than the ``Available To Process`` folder, it should be quarantined for reprocessing. For definitions of PHI/PII categories available for recaction, refer to the PHI/PII Definitions and Types section of these usage instructions below.
```
Comment (manthey) [inline docs/USAGE.rst:128]: Do you want to make this an rst link?  ``via Squish (`https://www.squishlist.com/seerdms/support/ <https://www.squishlist.com/seerdms/support/>`__)``

Should the `PHI/PII Definitions and Types section` also be a link? `` `PHI/PII Definitions and Types section <#phipii-definitions-and-types>`__``?

File: docs/USAGE.rst [hunk 2]
```diff
@@ -192,7 +192,7 @@ A folder in Girder may contain items, and an item always has to be in a folder.
    :width: 200
    :alt: image folder view
 
-To see an item view of an image, click on the image/item's row in the folder view. You will then go to the item view, which looks like the below screenshot, of an item named ``0579XY112001_01_13.svs`` that is located in the ``AvailableToProcess`` folder. In the info panel you can see some metadata such as the image size and WSI DeID creation date. The item view will present you with subsections for a panning/zooming ``Image Viewer``, a listing of ``Large Image Metadata``, the set of ``Associated Images``, and image/item specific ``WSI DeID Workflow`` actions.
+To see an item view of an image, click on the image/item's row in the folder view. You will then go to the item view, which looks like the below screenshot, of an item (a WI file) named ``0579XY112001_01_13.svs`` that is located in the ``AvailableToProcess`` folder. In the info panel, you can see some metadata such as the image size and WSI DeID creation date. The item view will present you with subsections for a panning/zooming ``Image Viewer``, a listing of ``Large Image Metadata``, the set of ``Associated Images``, and image/item specific ``WSI DeID Workflow`` actions.
```
Comment (manthey) [inline docs/USAGE.rst:195]: s/WI file/WSI file/

File: docs/USAGE.rst [hunk 3]
```diff
@@ -292,9 +292,9 @@ Review Process for PHI/PII in Image(s) and Metadata
  
 1. Open the WSI file, either by clicking on a specific Item or clicking the ``Next Item`` action.
 
-2. Review the image(s) and metadata fields for PHI/PII.
+2. Review the image(s) and metadata fields for PHI/PII.Definitions of PHI/PII categories in the redaction drop-down menu are listed in the PHI/PII Definitions and Types section below.
```
Comment (manthey) [inline docs/USAGE.rst:295]: Missing space.  Same link question as before.

**Comment by mgrauer** [comment]
@manthey Thanks for the QA. Looks good now?

**Review by manthey — APPROVED** [review]

**Pull request merged**